pact/ledger.pact

```diff
@@ -69,7 +69,7 @@
     @event true
   )
 
-  (defcap TOKEN:bool (id:string)
+  (defcap TOKEN:bool (id:string precision:integer policy:module{kip.token-policy-v1})
```

Comment:
This cap is defined in `kip.poly-fungible-v2` so we'll need to migrate it to `implements kip.poly-fungible-v3`